scripts/docker-compose-mysql/README.md

```diff
@@ -1 +0,0 @@
-Looking for the MySQL docs? Checkout the [Cromwell documentation](https://cromwell.readthedocs.io/en/develop/Configuring/#database)!
```

Comment:
The deleted files in `scripts/` required a Java update, but to my knowledge have not been used in years, so I decided to get rid of them.